transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/logging/Logging.java

```diff
@@ -14,13 +14,26 @@
 
 package com.google.android.datatransport.runtime.logging;
 
+import android.os.Build;
 import android.util.Log;
 
 public final class Logging {
   private Logging() {}
 
   private static String getTag(String tag) {
-    return "TransportRuntime." + tag;
+    String prefix = "TRuntime.";
```

Comment:
for convenience make this a class level constant

Reply:
Got it